Front-End/Javascript/Javascript-Basics/alertPromptConfirm.js

```diff
@@ -0,0 +1,11 @@
+//This just gives a popup with the entered message
+alert("Wait a minute who are you?");
+
+//This checks the response from the user and acts accordingly. It will give user two option "okay" or "cancel"
+let userResponse = confirm("Do you agree with company policies?");
```

Comment:
Asks the question with two possible answers and returns "true" or "false" based on the user selection